guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,72 @@
+To configure this, you can enable _dependency solving_.
 
-Dependency solving is relevant only in limited contexts.
-In the most common use cases, such as adding a security errata to a Content View with incremental update, dependency solving is likely unnecessary and can cause significant delays to Content View publication.
+For example, dependency solving is useful when you incrementally add a single package to a Content View version.
+You might need to enable dependency solving to install that package.
 
-The package dependency resolution feature does not take into account which packages are installed on your clients.
-Dependency solving considers only packages within the repositories of the Content View.
+However, dependency solving is unnecessary in most situations.
+For example:
+
+* When incrementally adding a security errata to a Content View, dependency solving can cause significant delays to Content View publication without major benefits.
+* Packages from a newer erratum might have dependencies that are incompatible with packages from an older Content View version.
+Incrementally adding the erratum using dependency solving might include unwanted packages.
+As an alternative, consider updating the Content View.
+
+[NOTE]
+====
+Dependency solving only considers packages within the repositories of the Content View.
+It does not consider packages installed on clients.
 ifeval::["{client-os-family}" == "Red Hat"]
-For example, if a Content View has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
+For example, if a Content View includes only AppStream, dependency solving does not include dependent BaseOS content at publish time.
 endif::[]
+
 For more information, see {ContentManagementDocURL}Limitations_to_Repository_Dependency_Resolution_content-management[Limitations to Repository Dependency Resolution] in _{ContentManagementDocTitle}_.
+====
 
-.Resolving Package Dependencies and Filters
-Filters, by default, do not resolve any dependencies of the packages listed in the filters.
-This might require some level of testing to determine what dependencies are required.
+Dependency solving can lead to the following problems:
 
-If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
+Significant delay in Content View publication::
+{Project} examines every repository in a Content View for dependencies.
+Therefore, publish time increases with more repositories.
++
+To mitigate this problem, use multiple Content Views with fewer repositories and combine them into composite Content Views.
 
-If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+Ignored content filters on dependent packages::
```

Comment:
```suggestion
Ignored Content View filters on dependent packages::
```
Terminological nitpick, but pls better verify with a SME.

Reply:
@ianballou Can you please take a look here? Can we go with Lennonka's suggestion?

Reply:
"Content view filters" is better, I agree there.